lua/astrocommunity/pack/typescript/typescript.lua

```diff
@@ -36,25 +36,23 @@ return {
 
       if not opts.handlers then opts.handlers = {} end
 
+      local has_prettier = function(util)
+        return util.root_has_file ".prettierrc"
+          or util.root_has_file ".prettierrc.json"
+          or util.root_has_file ".prettierrc.js"
+      end
+      local has_eslint = function(util) return util.root_has_file ".eslintrc.json" or util.root_has_file ".eslintrc.js" end
```

Comment:
should probably look for all expected files from their documentation https://eslint.org/docs/latest/use/configure/configuration-files#configuration-file-formats